routes/areas.js

```diff
@@ -10,6 +10,45 @@ Do not think we need a get areas by year because can just
 use the get all areas request then filter based on year on frontend
 */
 
+// get an area by id
+router.get('/area-management', async (req, res) => {
+  try {
+    const areaInfo = await pool.query(`
+      SELECT area.*, site.num_sites, site.site_info, mt_info.num_mts, student_info.num_students
+      FROM area
+        LEFT JOIN
+          (SELECT site.area_id, COUNT(site.site_id)::int as num_sites, array_agg(json_build_object('site_id', site.site_id, 'site_name', site.site_name)) as site_info
+          FROM site
+          GROUP BY site.area_id)
+        AS site ON site.area_id = area.area_id
+        LEFT JOIN
+          (SELECT site.area_id, COUNT(*)::int as num_mts
+          FROM master_teacher_site_relation as mt
+            INNER JOIN
+              (SELECT site.site_id, site.area_id FROM site)
+            AS site on site.site_id = mt.site_id
```

Comment:
sorry one more thing - it looks like the count of MTs is the number of rows in the MT/site relation where site id matches. However, not every teachers can be assigned to multiple sites. Can you count only distinct MTs when site id matches?

Reply:
i don't believe this problem exists with counting the number of sites and students. students can only be assigned to one site, and all sites will be unique anyways.

Reply:
whoops good catch! should be fixed now